teams.md/docs/csharp/getting-started/running-in-teams.md

```diff
@@ -25,45 +25,45 @@ To configure your agent for Teams, run the following command in the terminal ins
 :::
 
 ```bash
-npx @microsoft/teams.cli config add ttk.basic
+npx @microsoft/teams.cli config add atk.basic
 ```
 
 :::tip
-The `ttk.basic` configuration is a basic setup for Teams Toolkit. It includes the necessary files and configuration to get started with Teams development.<br/>
+The `atk.basic` configuration is a basic setup for Agents Toolkit. It includes the necessary files and configuration to get started with Teams development.<br/>
 Explore more advanced configurations as needed with teams config --help.<br />
 :::
 
-This [CLI](/developer-tools/cli) command adds configuration files required by Teams Toolkit, including:
+This [CLI](/developer-tools/cli) command adds configuration files required by Agents Toolkit, including:
 
 - Environment setup in the `env` folder and root `.env` file
 - Teams app manifest in the `appPackage` folder (if not already present)
 - Debug instructions in `.vscode/launch.json` and `.vscode/tasks.json`
-- TTK automation files to your project (e.g. `teamsapp.local.yml`)
+- ATK automation files to your project (e.g. `teamsapp.local.yml`)
 
 :::note
```

Comment:
This note seems to make things confusing.  The reader has to refer to several different products to understand the significance of this note.  Not sure if it is helpful here.

Reply:
fixed